src/mock/quic_p2p/node.rs

```diff
@@ -163,16 +163,16 @@ impl Node {
                         peer: Peer::new(peer_type, src),
                     })
                 }
+                self.send_pending_messages(src);
```

Comment:
Why is it outside the if?

Reply:
Same reason as in `ConnectSuccess` - we don't want this to get blocked even if we already considered ourselves connected. Worst case scenario, we already sent all pending messages and there is nothing to be sent again.

Reply:
We agreed on another patch - I'll resolve the conversation.